docs/src/pages/[platform]/connected-components/liveness/react-props.ts

```diff
@@ -74,21 +74,6 @@ export const FACE_LIVENESS_DETECTOR_COMPONENTS = [
   },
 ];
 
-export const FACE_LIVENESS_DETECTOR_CONFIG = [
```

Comment:
Just so i know what I'm approving, are we removing these props from the Liveness API?

Reply:
Nope, we just re-wrote them into the .mdx file directly because they did not conform to the reactProptsTable component style - https://github.com/aws-amplify/amplify-ui/pull/4106/files#diff-351796e5a623b829c355941ae08e4b9cb7807f74dd3924fca010100a1f629275L23